### DISCUSSION TITLE
tamagui can support New Architecture  ?

### DISCUSSION THREAD

**Discussion by ghost** [tamagui/tamagui#3064] [General]
tamagui can support New Architecture  ? with expo 52 and react native 0.76

**Comment by benjipott** [comment]
@ikaasraa not fully, the dialog and somecomponents like Select should be fixed

**Reply by biapar** [reply]
Now?

**Reply by flexbox** [reply]
Expo SDK 53 is using the new arch by default 
https://expo.dev/changelog/sdk-53#the-new-architecture-is-now-default-everywhere

the SDK was updated in tamagui 10 days ago
https://github.com/tamagui/tamagui/pull/3424

So we should be good

**Comment by flexbox** [comment]
@ikaasraa 
Yes https://github.com/tamagui/tamagui/discussions/3064#discussioncomment-13131573
